src/index.html

```diff
@@ -12,6 +12,146 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="forms">
+        <legend>Personal imformation</legend>
+        <div class="form-field">
+          <label for="Surname">Surname:
+            <input 
+                type="text" 
+                name="Surname"
+                autocomplete="off" 
+                id="Surname"
+            >
```

Comment:
```suggestion
            <input 
              type="text" 
              name="Surname"
              autocomplete="off" 
              id="Surname"
            >
```
Забагато відступів. Має бути 2